### ISSUE TITLE
Team Creation API fails

### ISSUE THREAD

**Issue by tlawrie** [boomerang-io/roadmap#230]
**Describe the bug**
The Team Creation API fails with
```
[2021-09-29 03:37:07.233] [ERROR] [     http-nio-8080-exec-3] [o.a.j.l.DirectJDKLog     ] : Servlet.service() for servlet [dispatcherServlet] in context with path [] threw exception [Request processing failed; nested exception is java.lang.NumberFormatException: For input string: "25Gi"] with root cause

java.lang.NumberFormatException: For input string: "25Gi"
	at java.base/java.lang.NumberFormatException.forInputString(NumberFormatException.java:65)
	at java.base/java.lang.Integer.parseInt(Integer.java:652)
	at java.base/java.lang.Integer.valueOf(Integer.java:983)
	at io.boomerang.service.crud.TeamServiceImpl.createStandaloneTeam(TeamServiceImpl.java:184)
	at io.boomerang.controller.api.TeamsV1Controller.addTeam(TeamsV1Controller.java:85)
```

**To Reproduce**
POST this body through to `/apis/v1/teams`

```
{
  "isActive": true,
  "name": "Tyson Team",
  "settings": {
    "properties": [
    ]
  },
  "quotas": {
    "maxWorkflowCount": 50,
    "maxWorkflowExecutionMonthly": 200,
    "maxWorkflowStorage": 25,
    "maxWorkflowExecutionTime": 120,
    "maxConcurrentWorkflows": 5
  },
  "approverGroups": [
  ],
  "users": [
  ],
  "workflows": [
  ]
}
```

**Expected behavior**
Team would be created

**Comment by tlawrie** [comment]
Its also partly because the default includes the "Gi" string I imagine.

**Comment by amhudson** [comment]
fixed with 3.1.54
